index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link href='https://fonts.googleapis.com/css?family=Poppins' rel='stylesheet'>
+    <link rel="stylesheet" href="./styles.css">
+    <title>Portfolio | Debabrata Mandal</title>
+</head>
+
+<body>
+    <section>
+        <header>
+            <div id="logo">Mr. Mandal</div>
+            <img id="menu-icon" alt="menu icon" src="./images/menu_icon.svg">
+        </header>
+        <article>
+            <header>
+                I'm Debabrata Mandal <br> Glad to see you!
+            </header>
+            <div>
+                I'm a software developer! I can help you build a product , feature or website Look through some of my work
+                and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </div>
+            <footer>
+                LET'S CONNECT
```

Comment:
- [ ] [Optional] Kindly wrap the "let's connect" in a `hx (h2, h3, ..., h6)` tag